iset.mm

```diff
@@ -65764,6 +65785,167 @@ elements or fails to hold (is equal to ` (/) ` ) for some element.
       cvv dom2 ) BCAADZEFBGZHZCGZHZQTAIZSEUATAJTAKLUDUAUCMTUBMNUBAITUBAOPR $.
   $}
 
+  ${
+    $d A x $.  $d B x $.  $d ph x $.
+    exmidfodomrlemeldju.a $e |- ( ph -> A C_ 1o ) $.
+    exmidfodomrlemeldju.el $e |- ( ph -> B e. ( A |_| 1o ) ) $.
+    $( Lemma for ~ exmidfodomr .  A variant of ~ djur .  (Contributed by Jim
+       Kingdon, 2-Jul-2022.) $)
+    exmidfodomrlemeldju $p |- ( ph
+        -> ( B = ( inl ` (/) ) \/ B = ( inr ` (/) ) ) ) $=
+      ( vx cinl cfv wceq wrex c0 cinr wo c1o wa wcel sylib fveq2d eqeq2d biimpd
+      cv sselda el1o rexlimdva imp orcd simpr olcd cdju djur syl mpjaodan ) ACF
+      UAZGHZIZFBJZCKGHZIZCKLHZIZMCUMLHZIZFNJZAUPOURUTAUPURAUOURFBAUMBPOZUOURVDU
+      NUQCVDUMKGVDUMNPZUMKIZABNUMDUBUMUCZQRSTUDUEUFAVCOUTURAVCUTAVBUTFNAVEOZVBU
+      TVHVAUSCVHUMKLVHVEVFAVEUGVGQRSTUDUEUHACBNUIPUPVCMEFBNCUJUKUL $.
+  $}
+
+  ${
+    $d A x $.  $d B x $.  $d ph x $.
+    exmidfodomrlemreseldju.a $e |- ( ph -> A C_ 1o ) $.
+    exmidfodomrlemreseldju.el $e |- ( ph -> B e. ( A |_| 1o ) ) $.
+    $( Lemma for ~ exmidfodomrlemrALT .  A variant of ~ eldju .  (Contributed
+       by Jim Kingdon, 9-Jul-2022.) $)
+    exmidfodomrlemreseldju $p |- ( ph
+        -> ( ( (/) e. A /\ B = ( ( inl |` A ) ` (/) ) )
+          \/ B = ( ( inr |` 1o ) ` (/) ) ) ) $=
+      ( vx cres cfv wceq wrex c0 wcel wa c1o sylib fveq2d eqeq2d simpr biimpd
+      wo cinl cinr sselda el1o eqeltrrd biantrurd bitrd rexlimdva imp orcd olcd
+      cv cdju eldju mpjaodan ) ACFULZUABGZHZIZFBJZKBLZCKUQHZIZMZCKUBNGZHZIZTCUP
+      VEHZIZFNJZAUTMVDVGAUTVDAUSVDFBAUPBLZMZUSVDVLUSVCVDVLURVBCVLUPKUQVLUPNLZUP
+      KIZABNUPDUCUPUDZOZPQVLVAVCVLUPKBVPAVKRUEUFUGSUHUIUJAVJMVGVDAVJVGAVIVGFNAV
+      MMZVIVGVQVHVFCVQUPKVEVQVMVNAVMRVOOPQSUHUIUKACBNUMLUTVJTEFBNCUNOUO $.
+  $}
+
+  ${
+    $d f g x z $.  $d f g y z $.  $d g u x $.  $d g v x $.  $d w z $.
+    $( Excluded middle implies the existence of a function onto an inhabited
```

Comment:
an --> any ?

Plus, I'm not sure if it's me or if the grammar of theses sentences is weird. A correct formulation would be much heavier, so I guess this is ok.

Reply:
The grammar of these sentences is definitely weird. I tried again starting from the set.mm comment for fodomr and came up with:

```
Excluded middle is equivalent to the existence of 
a mapping from any set onto any inhabited set that it
dominates
```

which only changes a few words but which I think is better.

Reply:
Perfect !